spec/spec_helper_acceptance_local.rb

```diff
@@ -43,6 +43,10 @@ def update_profile_file
   run_shell("sed -i '/mesg n || true/c\\test -t 0 && mesg n || true' ~/.profile")
 end
 
+def get_os_name # rubocop:disable Style/AccessorMethodName: Do not prefix reader method names with get_
```

Comment:
Would recommend using `fetch_` instead of `get_` prefix on any operation that'll take a while.

Reply:
done! 👍